packages/vscode-extension/src/commonlib/azureLogin.ts

```diff
@@ -242,12 +243,27 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
         "SignOut"
       );
     }
-    await vscode.commands.executeCommand("azure-account.logout");
-    AzureAccountManager.tenantId = undefined;
-    AzureAccountManager.subscriptionId = undefined;
-    return new Promise((resolve) => {
-      resolve(true);
-    });
+    try {
+      await vscode.commands.executeCommand("azure-account.logout");
+      AzureAccountManager.tenantId = undefined;
+      AzureAccountManager.subscriptionId = undefined;
+      ExtTelemetry.sendTelemetryEvent(TelemetryEvent.SingOut, {
+        [TelemetryProperty.AccountType]: AccountType.Azure,
+        [TelemetryProperty.Success]: TelemetrySuccess.Yes
+      });
+      return new Promise((resolve) => {
+        resolve(true);
+      });
+    } catch (e) {
+      VsCodeLogInstance.error(
+        "[Logout Azure] " + e.message
+      );
+      ExtTelemetry.sendTelemetryEvent(TelemetryEvent.SingOut, {
```

Comment:
sendTelemetryErrorEvent instead of sendTelemetryEvent.

Reply:
Fixed.